lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -346,26 +358,21 @@ private Map<String, Object> buildNoMatchResponse(
 
             return JOURNEY_RESET_IDENTITY;
         }
-        LOGGER.info(LogHelper.buildLogMessage("New user so returning next."));
+        LOGGER.info(LogHelper.buildLogMessage("New IPV journey required, returning journey next"));
         return JOURNEY_NEXT;
     }
 
-    private Map<String, Object> buildProfileMatchedResponse(
-            Gpg45Profile matchedProfile, AuditEventUser auditEventUser) throws SqsException {
-        var message =
-                new StringMapMessage()
-                        .with(
-                                LOG_MESSAGE_DESCRIPTION.getFieldName(),
-                                "Matched profile and within CI threshold so returning reuse journey.")
-                        .with(LOG_PROFILE.getFieldName(), matchedProfile.getLabel());
-        LOGGER.info(message);
-
-        auditService.sendAuditEvent(
-                new AuditEvent(
-                        AuditEventTypes.IPV_IDENTITY_REUSE_COMPLETE,
-                        configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
-                        auditEventUser));
+    private Map<String, Object> buildReuseResponse(Vot vot, AuditEventUser auditEventUser)
+            throws SqsException {
+        if (vot.hasGpg45Profiles()) {
```

Comment:
This is an interesting question - we'll have to determine whether we still want this audit event for reuse of an operational profile. Perhaps worth asking Simon Parsons?

It might become less relevant once we've got our journey split, and we have a new explicit event.